### PR TITLE
fix: Update provider file to use only gnap provider

### PR DESCRIPTION
**Pull request by talwinder50**

### CODE REVIEW

**Review by birtony — COMMENTED** [review]
This would be a breaking change. Can we update automation tests in wallet first to make sure all functionality works, before removing the ZCAP flow mock bank?

File: test/bdd/fixtures/auth-rest/oidc-config/providers.yaml [hunk 1]
```diff
@@ -6,18 +6,6 @@
...
-    clientID: auth
-    clientSecret: auth-secret
-    name: Demo OIDC
-    signUpIconURL:
-      en: https://localhost:8070/static/images/sign-up-icon-en.svg
-      fr: https://localhost:8070/static/images/sign-up-icon-fr.svg
-    signInIconURL:
-      en: https://localhost:8070/static/images/sign-in-icon-en.svg
-      fr: https://localhost:8070/static/images/sign-in-icon-fr.svg
-    order: 1
```
Comment (birtony) [inline test/bdd/fixtures/auth-rest/oidc-config/providers.yaml:19]: order should be `1`

Comment (talwinder50) [reply test/bdd/fixtures/auth-rest/oidc-config/providers.yaml:19]: updatted

**Review by birtony — APPROVED** [review]

**Review by rolsonquadras — CHANGES_REQUESTED** [review]
Auth service checks both OIDC/zcap as well as GNAP - hence we have 2 providers. The real changes should be in Wallet repo and not here.

**Comment by talwinder50** [comment]
> Auth service checks both OIDC/zcap as well as GNAP - hence we have 2 providers. The real changes should be in Wallet repo and not here.

Okay will close this pr and update in wallet only then

**Pull request closed without merging**